src/components/map/MapComponent.vue

```diff
@@ -50,7 +50,9 @@ const viewStore = useViewsStore()
 let grid
 
 onMounted(async () => {
-  await rennesApp.initializeMap()
+  if (viewStore.currentView !== 'step-sunshine') {
```

Comment:
Because, when the user finishes his simulation and go to AutocalsolView, if he want to goback and cancel the simulation, it allows him to see his house highlighted again. Without that, the map reinitialize and the map view is the same as home page.